test/langtools/tools/javac/patterns/withers/Model.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
probably this test should be in another location not under `patterns`, same for other tests added inside `patterns` folder